packages/oidc-mock-provider/src/index.ts

```diff
@@ -102,7 +107,10 @@ export class OIDCMockProvider {
   }
 
   private async init(): Promise<this> {
-    this.httpServer.listen(this.config.port ?? 0, this.config.hostname);
+    this.httpServer.listen(
+      this.config.port ?? 0,
+      this.config.bindIpAll ? '::,0.0.0.0' : this.config.hostname
```

Comment:
> `Uncaught Error: getaddrinfo ENOTFOUND ::,0.0.0.0`

This should probably be/can just be

```suggestion
      this.config.bindIpAll ? '::' : this.config.hostname
```